packages/compiler/test/selector/selector_spec.ts

```diff
@@ -126,6 +126,91 @@ import {el} from '@angular/platform-browser/testing/src/browser_util';
       expect(matched).toEqual([s1[0], 1]);
     });
 
+    it('should support "$" in attribute names', () => {
+      matcher.addSelectables(s1 = CssSelector.parse('[someAttr$]'), 1);
+
+            expect(
```

Comment:
Does the formatter complain with:

```suggestion
      expect(
```